dev_guide/getting_traffic_into_cluster.adoc

```diff
@@ -0,0 +1,147 @@
+[[getting-traffic-into-cluster]]
+= Getting Traffic Into The Cluster
+{product-author}
+{product-version]
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+ifdef::openshift-origin,openshift-enterprise,openshift-dedicated[]
+== Overview
+There are many ways to access the cluster. This section describes some
+commonly used approaches.
+
+The recommendation is:
+
+- If you have HTTP/HTTPS -- Use the xref:../dev_guide/getting_traffic_into_cluster.adoc#using-a-router[Router],
+- If you have Encrypted TCP (i.e., TLS with the SNI header) -- Use the xref:../dev_guide/getting_traffic_into_cluster.adoc#using-a-router[Router],
+- If you have TCP or UDP -- Use xref:../dev_guide/getting_traffic_into_cluster.adoc#using-the-loadbalancer[LoadBalancer],
+xref:../dev_guide/getting_traffic_into_cluster.adoc#using-externalIP[ExternalIP], or
+xref:../dev_guide/getting_traffic_into_cluster.adoc#using-nodeport[NodePort].
+
+
+The admin configures DNS or wildcard DNS records to point to IP addresses
```

Comment:
This is somewhat confusing.  We just need to say that the admin is in charge of setting this up... not us.  And they need to make sure that the DNS name lands on an IP they are exposing.  Noting they can use a wildcard is good, but we should say they may want to use a wildcard so that the users can self-provision routers.  For the other ingress methods, a wildcard is less useful.